code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -597,11 +597,19 @@
 /datum/reagent/consumable/eggyolk
 	name = "Egg Yolk"
 	description = "It's full of protein."
-	nutriment_factor = 3 * REAGENTS_METABOLISM
+	nutriment_factor = 4 * REAGENTS_METABOLISM
```

Comment:
i would expect the 3 to be split around.

just thinking from an egg if you took this prior to PR you get 3*RM, now if you take from an egg post PR you get 5.5*RM

does it matter? i have no idea i d ont balance food.